--- "a/HandyNotes_WarfrontRares/HandyNotes_WarfrontRares.lua"
+++ "b/HandyNotes_WarfrontRares/HandyNotes_WarfrontRares.lua"
@@ -1046,7 +1046,7 @@ function Arathi:ShowNode( mapFile, node )
 	if ( MYFACTION ~= nil and node["faction"] ~= nil and node["faction"] ~= MYFACTION ) then return false end
 	for i,q in pairs(node["questId"]) do
 		-- print( node["questId"][i] );
-		if ( IsQuestFlaggedCompleted( node["questId"][i] ) ) then return false end
+		if ( C_QuestLog.IsQuestFlaggedCompleted( node["questId"][i] ) ) then return false end
 	end
     return true
 end
