--- "a/MTLove/mtlove-core/MTLoveLocalization.lua"
+++ "b/MTLove/mtlove-core/MTLoveLocalization.lua"
@@ -24,7 +24,7 @@ MTLove_GUI_License									= MTLove_get_globalValues("UI_Name") .. " is free sof
 MTLove_UI_OnLoad									= MTLove_get_globalValues("UI_Name") .. " v" .. MTLove_get_globalValues("UI_Version") .. " by Herrmann, Tom from Randoom.org, edited by malfdawg.";
 MTLove_UI_Releaseversion							= "\nPlease send your feedback to " .. MTLove_get_globalValues("UI_Mail") .. " and/or visit the project page at " .. MTLove_get_globalValues("UI_Website");
 MTLove_UI_Testversion								= "\nPlease report any bugs of this test version to " .. MTLove_get_globalValues("UI_Mail") .. " and visit " .. MTLove_get_globalValues("UI_Website") .. " for updates.";
-MTLove_UI_Variables_Loaded							= MTLove_UI_CMD_prefix .. "Settings for " .. UnitName("player") .. " of " .. GetCVar("realmName") .. " loaded.";
+MTLove_UI_Variables_Loaded							= MTLove_UI_CMD_prefix .. "Settings for " .. UnitName("player") .. " of " .. GetRealmName() .. " loaded.";
 MTLove_UI_Defaults									= MTLove_UI_CMD_prefix .. "Defaults for all settings loaded.";
 MTLove_UI_OldVersionDefaults						= MTLove_UI_CMD_prefix .. "Previous version too old for settings conversion. Defaults for all settings loaded.";
 MTLove_UI_GarbageClean								= MTLove_UI_CMD_prefix .. "You recently updated " .. MTLove_get_globalValues("UI_Name") .. ".\n" .. MTLove_UI_CMD_prefix .. "Your settings were cleaned.";
