diff --git "a/mtlove-core/MTLove.xml" "b/mtlove-core/MTLove.xml"
index 8077807..bbc9904 100644
--- "a/mtlove-core/MTLove.xml"
+++ "b/mtlove-core/MTLove.xml"
@@ -39,24 +39,16 @@
 		</Scripts>
 	</Frame>
 
-	<Frame name="MTLove_Frame" hidden="true" frameStrata="TOOLTIP" parent="UIParent">
+	<Frame name="MTLove_Frame" hidden="true" frameStrata="TOOLTIP" parent="UIParent" inherits="BackdropTemplate">
+		<KeyValues>
+			<KeyValue key="backdropInfo" value="BACKDROP_TOOLTIP_16_16_5555" type="global"/>
+		</KeyValues>
 		<Size>
 			<AbsDimension x="0" y="28"/>
 		</Size>
 		<Anchors>
 			<Anchor point="BOTTOMCENTER" />
 		</Anchors>
-		<Backdrop bgFile="Interface\Tooltips\UI-Tooltip-Background" edgeFile="Interface\Tooltips\UI-Tooltip-Border" tile="true">
-			<EdgeSize>
-				<AbsValue val="16"/>
-			</EdgeSize>
-			<TileSize>
-				<AbsValue val="16"/>
-			</TileSize>
-			<BackgroundInsets>
-				<AbsInset left="5" right="5" top="5" bottom="5"/>
-			</BackgroundInsets>
-		</Backdrop>
 		<Layers>
 			<Layer level="OVERLAY">
 				<FontString name="$parent_Line0" inherits="GameTooltipHeaderText" justifyV="CENTER" justifyH="CENTER">
@@ -70,7 +62,10 @@
 			</Layer>
 		</Layers>
 		<Frames>
-			<Frame name="$parent_Counter" hidden="true" frameStrata="TOOLTIP">
+			<Frame name="$parent_Counter" hidden="true" frameStrata="TOOLTIP" inherits="BackdropTemplate">
+				<KeyValues>
+					<KeyValue key="backdropInfo" value="BACKDROP_TOOLTIP_16_16_5555" type="global"/>
+				</KeyValues>
 				<Size>
 					<AbsDimension x="0" y="23"/>
 				</Size>
@@ -81,17 +76,6 @@
 						</Offset>
 					</Anchor>
 				</Anchors>
-				<Backdrop bgFile="Interface\Tooltips\UI-Tooltip-Background" edgeFile="Interface\Tooltips\UI-Tooltip-Border" tile="true">
-					<EdgeSize>
-						<AbsValue val="16"/>
-					</EdgeSize>
-					<TileSize>
-						<AbsValue val="16"/>
-					</TileSize>
-					<BackgroundInsets>
-						<AbsInset left="5" right="5" top="5" bottom="5"/>
-					</BackgroundInsets>
-				</Backdrop>
 				<Layers>
 					<Layer level="OVERLAY">
 						<FontString name="$parent_Line0" inherits="GameTooltipText" justifyV="CENTER" justifyH="CENTER">
@@ -156,24 +140,16 @@
 		</Frames>
 	</Frame>
 
-  	<Frame name="MTLove_TT_Frame" hidden="true" frameStrata="TOOLTIP" parent="UIParent">
+  	<Frame name="MTLove_TT_Frame" hidden="true" frameStrata="TOOLTIP" parent="UIParent" inherits="BackdropTemplate">
+		<KeyValues>
+			<KeyValue key="backdropInfo" value="BACKDROP_TOOLTIP_16_16_5555" type="global"/>
+		</KeyValues>
 		<Size>
 			<AbsDimension x="0" y="48"/>
 		</Size>
 		<Anchors>
 			<Anchor point="BOTTOMCENTER" />
 		</Anchors>
-		<Backdrop bgFile="Interface\Tooltips\UI-Tooltip-Background" edgeFile="Interface\Tooltips\UI-Tooltip-Border" tile="true">
-			<EdgeSize>
-				<AbsValue val="16"/>
-			</EdgeSize>
-			<TileSize>
-				<AbsValue val="16"/>
-			</TileSize>
-			<BackgroundInsets>
-				<AbsInset left="5" right="5" top="5" bottom="5"/>
-			</BackgroundInsets>
-		</Backdrop>
 		<Layers>
 			<Layer level="OVERLAY">
 				<FontString name="$parent_Line0" inherits="GameTooltipHeaderText" justifyV="BOTTOM" justifyH="LEFT">
@@ -223,21 +199,13 @@
 			</StatusBar>
 		</Frames>
   	</Frame>
-	<Frame name="MTLove_TT_Frame_Counter" hidden="true" frameStrata="TOOLTIP" parent="UIParent">
+	<Frame name="MTLove_TT_Frame_Counter" hidden="true" frameStrata="TOOLTIP" parent="UIParent" inherits="BackdropTemplate">
+		<KeyValues>
+			<KeyValue key="backdropInfo" value="BACKDROP_TOOLTIP_16_16_5555" type="global"/>
+		</KeyValues>
 		<Size>
 			<AbsDimension x="0" y="23"/>
 		</Size>
-		<Backdrop bgFile="Interface\Tooltips\UI-Tooltip-Background" edgeFile="Interface\Tooltips\UI-Tooltip-Border" tile="true">
-			<EdgeSize>
-				<AbsValue val="16"/>
-			</EdgeSize>
-			<TileSize>
-				<AbsValue val="16"/>
-			</TileSize>
-			<BackgroundInsets>
-				<AbsInset left="5" right="5" top="5" bottom="5"/>
-			</BackgroundInsets>
-		</Backdrop>
 		<Layers>
 			<Layer level="OVERLAY">
 				<FontString name="$parent_Line0" inherits="GameTooltipText" justifyV="CENTER" justifyH="CENTER">
@@ -251,21 +219,13 @@
 			</Layer>
 		</Layers>
 	</Frame>
-	<Frame name="MTLove_TargetCounter_Counter" hidden="true" parent="TargetFrame">
+	<Frame name="MTLove_TargetCounter_Counter" hidden="true" parent="TargetFrame" inherits="BackdropTemplate">
+		<KeyValues>
+			<KeyValue key="backdropInfo" value="BACKDROP_TOOLTIP_16_16_5555" type="global"/>
+		</KeyValues>
 		<Size>
 			<AbsDimension x="0" y="23"/>
 		</Size>
-		<Backdrop bgFile="Interface\Tooltips\UI-Tooltip-Background" edgeFile="Interface\Tooltips\UI-Tooltip-Border" tile="true">
-			<EdgeSize>
-				<AbsValue val="16"/>
-			</EdgeSize>
-			<TileSize>
-				<AbsValue val="16"/>
-			</TileSize>
-			<BackgroundInsets>
-				<AbsInset left="5" right="5" top="5" bottom="5"/>
-			</BackgroundInsets>
-		</Backdrop>
 		<Layers>
 			<Layer level="OVERLAY">
 				<FontString name="$parent_Line0" inherits="GameTooltipText" justifyV="CENTER" justifyH="CENTER">
@@ -279,21 +239,13 @@
 			</Layer>
 		</Layers>
 	</Frame>
-	<Frame name="MTLove_FocusCounter_Counter" hidden="true" parent="FocusFrame">
+	<Frame name="MTLove_FocusCounter_Counter" hidden="true" parent="FocusFrame" inherits="BackdropTemplate">
+		<KeyValues>
+			<KeyValue key="backdropInfo" value="BACKDROP_TOOLTIP_16_16_5555" type="global"/>
+		</KeyValues>
 		<Size>
 			<AbsDimension x="0" y="23"/>
 		</Size>
-		<Backdrop bgFile="Interface\Tooltips\UI-Tooltip-Background" edgeFile="Interface\Tooltips\UI-Tooltip-Border" tile="true">
-			<EdgeSize>
-				<AbsValue val="16"/>
-			</EdgeSize>
-			<TileSize>
-				<AbsValue val="16"/>
-			</TileSize>
-			<BackgroundInsets>
-				<AbsInset left="5" right="5" top="5" bottom="5"/>
-			</BackgroundInsets>
-		</Backdrop>
 		<Layers>
 			<Layer level="OVERLAY">
 				<FontString name="$parent_Line0" inherits="GameTooltipText" justifyV="CENTER" justifyH="CENTER">
