--- "a/ChatBar Classic/ChatBar Classic.lua"
+++ "b/ChatBar Classic/ChatBar Classic.lua"
@@ -136,7 +136,7 @@ v1.2
 v1.1
 -Addon Channels Hidden added GuildMap
 -Text has been made Localizable
--Officer chat shows up if you CanEditOfficerNote()
+-Officer chat shows up if you C_GuildInfo.CanEditOfficerNote()
 -Buttons now correctly update when raid, party, and guild changes
 -Hide Text now correctly says Show Text
 -Fixed button for channel 8 to diplay and tooltip correctly
@@ -392,7 +392,7 @@ ChatBar_ChatTypes = {
 		blockable = true,
 		chatGroup = "GUILD_OFFICER",
 		show = function()
-			return CanEditOfficerNote() and (not ChatBar_HiddenButtons[CHAT_MSG_OFFICER]);
+			return C_GuildInfo.CanEditOfficerNote() and (not ChatBar_HiddenButtons[CHAT_MSG_OFFICER]);
 		end
 	},
 	{
