--- "a/tdBattlePetScript/Libs/tdGUI/Widget/DropMenu.lua"
+++ "b/tdBattlePetScript/Libs/tdGUI/Widget/DropMenu.lua"
@@ -30,6 +30,7 @@ function DropMenu:Constructor(_, style, withOtherMenu, menuList, level)
 
     if style == 'MENU' then
         self:SetPadding(15)
+		Mixin(DropMenu, BackdropTemplateMixin)
         self:SetBackdrop{
             bgFile = [[Interface\Tooltips\UI-Tooltip-Background]],
             edgeFile = [[Interface\Tooltips\UI-Tooltip-Border]],

--- "a/tdBattlePetScript/UI/Widgets/AutoCompleteBox.lua"
+++ "b/tdBattlePetScript/UI/Widgets/AutoCompleteBox.lua"
@@ -22,6 +22,7 @@ function AutoCompleteBox:Constructor()
 
     self:SetPadding(13)
     self:SetItemSpacing(5)
+	Mixin(AutoCompleteBox, BackdropTemplateMixin)
     self:SetBackdrop{
         bgFile = [[Interface\Tooltips\UI-Tooltip-Background]],
         edgeFile = [[Interface\Tooltips\UI-Tooltip-Border]],
